recipe/meta.yaml

```diff
@@ -69,6 +69,7 @@ requirements:
     - {{ cdt('libxcomposite-devel') }}   # [linux]
     - {{ cdt('libxcursor-devel') }}      # [linux]
     - {{ cdt('libxi-devel') }}           # [linux]
+    - {{ cdt('libxinerama-devel') }}     # [linux]
```

Comment:
This is necessary since it appears as a private dependency of gtk3 (gdk3) inside the pkg-config file.... sigh... i don't get why they check for private dependencies too but that is just how pkg-config works.

I could alternatively use `sed` to remove the `RequiresPrivate` line and I think it will still work.